contracts/WalletSimple.sol

```diff
@@ -187,6 +187,8 @@ contract WalletSimple is IERC721Receiver, ERC1155Receiver {
     uint256 sequenceId,
     bytes calldata signature
   ) external onlySigner {
+    require(toAddress != address(0), 'Invalid destination address');
```

Comment:
I think validating zero address check shouldn't cause any issues on any evm chain, infact it is a good practice to have these checks to avoid loss of funds. Will add this check in batchTransfer and sendMultisigToken as well as it is missing in them

Reply:
Modified [this](https://github.com/BitGo/eth-multisig-v4/pull/120/commits/261b4f64d3614081230e7a356f2a59658ae9a521) commit